api/src/cli/commands/roles/create.ts

```diff
@@ -14,10 +16,32 @@ export default async function rolesCreate({ role: name, admin }: { role: string;
 
 	try {
 		const schema = await getSchema();
-		const service = new RolesService({ schema: schema, knex: database });
+		const rolesService = new RolesService({ schema: schema, knex: database });
+		const policiesService = new PoliciesService({ schema: schema, knex: database });
+		const accessService = new AccessService({ schema: schema, knex: database });
 
-		const id = await service.createOne(admin ? { name, admin_access: admin } : { name });
-		process.stdout.write(`${String(id)}\n`);
+		const adminPolicyId = await policiesService.knex
```

Comment:
We should use `readOne` from the service here, as there might be differences in how databases store the values under the hood (f.e. in SQLite a boolean is `0` / `1` and will fail with a "where `true`"). The Directus methods will type cast these differences away.

That being said, seeing you're trying to create a whole new role, should we instead opt to just create a new admin policy that's just for this role? 🤔 Might be less confusing than reusing one that might be named for a certain use case or group.